test/unit/specs/methods.spec.js

```diff
@@ -69,10 +69,27 @@ describe('ti.map', function () {
 			// win.add(view);
 			// win.open();
 		});
+		if (!ANDROID) {
+		// TiMapImageOverlayProxy
+			describe('setImage', () => {
+				it('to set image on Map',  () => {
+					var fileSystem = Ti.Filesystem.getFile(Ti.Filesystem.resourcesDirectory, 'images', 'flower.jpg.keep');
+					var newBlob = fileSystem.read();
+
+					expect(Map.setImage(newBlob)).toEqual(jasmine.any(Function));
+				});
+			});
 
-		// TDOO: Test actually adding an annotation to a view
+			describe('setBoundsCoordinate', () => {
+				it('to set bound coordinates',  () => {
+					var fileSystem = Ti.Filesystem.getFile(Ti.Filesystem.resourcesDirectory, 'images', 'flower.jpg.keep');
+					var newBlob = fileSystem.read();
+
+					expect(Map.setBoundsCoordinate(newBlob)).toEqual(jasmine.any(Function));
+				});
+			});
+			// TDOO: Test actually adding an annotation to a view
```

Comment:
@anil-shukla-axway, did you verified this TODO?

Reply:
No I did not verified this.

Reply:
Please verify and and update PR if required.